rosdep/base.yaml

```
@@ -2871,6 +2871,9 @@ libfftw3:
   nixos: [fftw, fftwSinglePrec]
   openembedded: [fftw@meta-oe]
   ubuntu: [libfftw3-3, libfftw3-dev]
+libfl-dev:
+  debian: [libfl-dev]
```

Looks like the Fedora package underwent some naming changes, but it does exist: https://src.fedoraproject.org/rpms/flex#bodhi_updates

The automation is also indicating that openSUSE has this package: https://software.opensuse.org/package/flex

```suggestion
  debian: [libfl-dev]
  fedora:
    '*': [flex-devel]
    '33': [libfl-devel]
  opensuse: [libfl-devel]
```

---

@cottsay I'm curious, why single out 33? It looks like rpms are available for Fedora 34 and 35 as well.

---

The package name changed in 34. In 33 and earlier, it was called `flex-devel`, and was completely renamed to `libfl-devel` for Fedora 34 and later (covered by the `*`): https://src.fedoraproject.org/rpms/flex/c/9adf2296f3603e2ee68328283ad625d080ae1c2a?branch=rawhide

---

Also: Fedora 33 is the oldest non-EOL Fedora release right now.

---

Thanks, I didn't get that from the source package overview page!